lib/player.rb

```diff
@@ -0,0 +1,25 @@
+class Player 
+  include Interface
+
+  attr_reader :char
+  attr_accessor :name, :inputs
+
+  def initialize(name, inputs=[], char)
+    @name = name
+    @inputs = inputs
+    @char = char
+  end
+
+  def initial  
+    if @name.start_with?('player')
+      @name = getInput('name')
+    end
+    @inputs = []
+  end
+
+  def update(input)
```

Comment:
I would suggest changing the method name into something more descriptive of what we want our Player to do here. It appears here the Player is making his turn so I guess it should be more properly named `turn` or `move`

Reply:
done.